sagemaker-jumpstart/object_detection/JumpStart_Object_Detection.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        entry_point="inference.py",
</pre><p>Where is the inference.py script coming from? I don't see it in the PR or elsewhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>It is coming from source_dir. Added a comment for the same.</p>